src/modules/tests/test-data/cat-home/controlled-stop/__tests__/controlled-stop.reducer.spec.ts

```diff
@@ -0,0 +1,74 @@
+import { CatFUniqueTypes } from '@dvsa/mes-test-schema/categories/F';
```

Comment:
Can we not have a common controlled stop reducer for B and Home , objects are the same so it should be easy to achive.

Reply:
Happy for this to be chucked into the tech debt list - to get this in though

Reply:
Yeah, good shout. I will do it as a separate PR though as we would making a change to category in prod. Better traceability that way.